ERCS/erc-725.md

```diff
@@ -37,270 +37,249 @@ And the event:
 
 - `OwnershipTransferred(address indexed previousOwner, address indexed newOwner)`
 
----
-
-### `ERC725X`
+### Y
 
-**`ERC725X`** interface id according to [ERC-165](./eip-165.md): `0x7545acac`.
+**Y** interface id according to [ERC-165]: `0x629aa694`.
 
-Smart contracts implementing the `ERC725X` standard MUST implement the [ERC-165](./eip-165.md) `supportsInterface(..)` function and MUST support the `ERC165` and `ERC725X` interface ids.
+Smart contracts implementing the ERC-725 Y standard MUST implement the [ERC-165] `supportsInterface(..)` function and MUST support the ERC-165 and Y interface ids.
 
-### `ERC725X` Methods
+### Methods
```

Comment:
You should move your interface definitions out of the Security Considerations section to somewhere around here, in the Specification section.